diff --git a/src/condor_credd/condor_credmon_oauth/condor_credmon_vault b/src/condor_credd/condor_credmon_oauth/condor_credmon_vault
index 2ca146e260..57d8bb7c41 100755
--- a/src/condor_credd/condor_credmon_oauth/condor_credmon_vault
+++ b/src/condor_credd/condor_credmon_oauth/condor_credmon_vault
@@ -1,4 +1,4 @@
-#!/usr/bin/python3
+#!/usr/bin/env python3
 
 import os
 import time
diff --git a/src/condor_scripts/common-cloud-attributes-aws.py b/src/condor_scripts/common-cloud-attributes-aws.py
index 4e2ae135ea..2075ab5a51 100755
--- a/src/condor_scripts/common-cloud-attributes-aws.py
+++ b/src/condor_scripts/common-cloud-attributes-aws.py
@@ -1,4 +1,4 @@
-#!/usr/bin/python3
+#!/usr/bin/env python3
 
 import requests
 
diff --git a/src/condor_scripts/common-cloud-attributes-google.py b/src/condor_scripts/common-cloud-attributes-google.py
index 83005dc71d..d8ec573621 100755
--- a/src/condor_scripts/common-cloud-attributes-google.py
+++ b/src/condor_scripts/common-cloud-attributes-google.py
@@ -1,4 +1,4 @@
-#!/usr/bin/python3
+#!/usr/bin/env python3
 r"""Print HTCondor machine ClassAd attributes derived from Google Cloud Metadata Servers
 
 Converts Google Cloud instance metadata into HTCondor ClassAd attribute/value
diff --git a/src/condor_scripts/condor_check_config b/src/condor_scripts/condor_check_config
index b3bdf2a1df..5b02380241 100755
--- a/src/condor_scripts/condor_check_config
+++ b/src/condor_scripts/condor_check_config
@@ -1,4 +1,4 @@
-#!/usr/bin/python3
+#!/usr/bin/env python3
 
 """
 Scan through the various HTCondor config knobs looking for potential
@@ -25,7 +25,7 @@ def format_red_terminal_text(text):
 
 def check_allow_daemon():
     """
-    Look for missing ALLOW_DAEMON and ALLOW_ADVERTISE_* 
+    Look for missing ALLOW_DAEMON and ALLOW_ADVERTISE_*
     At least one of which is needed for the collector
     """
 
diff --git a/src/condor_scripts/condor_check_password b/src/condor_scripts/condor_check_password
index 944757bbe8..8abe60fd98 100755
--- a/src/condor_scripts/condor_check_password
+++ b/src/condor_scripts/condor_check_password
@@ -1,4 +1,4 @@
-#!/usr/bin/python3
+#!/usr/bin/env python3
 
 """
 Scan through the various HTCondor signing keys and passwords, looking
@@ -25,7 +25,7 @@ def format_red_terminal_text(text):
     if g_color_output:
         return "\033[1m\033[91m%s\033[0m" % text
     return text
-        
+
 
 def simple_scramble(in_buf):
     """
@@ -52,7 +52,7 @@ def truncate_at_null(in_buf):
 
 def atomic_write_contents(dest, contents):
     """
-    Atomically write `contents` to Path `dest` and fsync the result. 
+    Atomically write `contents` to Path `dest` and fsync the result.
     """
     with tempfile.NamedTemporaryFile(mode="w+b", prefix=".condor." + dest.name, dir=dest.parent, delete=False) as tf:
         tf.write(contents)
diff --git a/src/condor_scripts/condor_watch_q b/src/condor_scripts/condor_watch_q
index a95aa82f2d..2a3e540ff1 100755
--- a/src/condor_scripts/condor_watch_q
+++ b/src/condor_scripts/condor_watch_q
@@ -1,4 +1,4 @@
-#!/usr/bin/python3
+#!/usr/bin/env/python3
 
 # Copyright 2020 HTCondor Team, Computer Sciences Department,
 # University of Wisconsin-Madison, WI.
@@ -57,7 +57,7 @@ def parse_args():
             If no users, cluster ids, event logs, or batch names are given,
             condor_watch_q will default to tracking all of the current user's jobs.
 
-            By default, condor_watch_q will never exit on its own 
+            By default, condor_watch_q will never exit on its own
             (unless it encounters an error or it is not tracking any jobs).
             You can tell it to exit when certain conditions are met. For example,
             to exit with status 0 when all of the jobs it is tracking are done
@@ -121,7 +121,7 @@ def parse_args():
         metavar="GROUPER,JOB_STATUS[,EXIT_STATUS]",
         help=textwrap.dedent(
             """
-            Specify conditions under which condor_watch_q should exit. 
+            Specify conditions under which condor_watch_q should exit.
 
             GROUPER is one of {{{exit_groupers}}}.
 
